src/sentry/static/sentry/app/components/events/errorLevel.tsx

```diff
@@ -1,8 +1,9 @@
 import styled from '@emotion/styled';
+import {Level} from 'app/types';
 
 const DEFAULT_SIZE = '13px';
 
-function getLevelColor({level, theme}) {
+function getLevelColor({level = '', theme}) {
```

Comment:
Looks like this is missing types too? `theme` doesn't have a type does it?, and shouldn't level also have a type?